cpp/src/table/row_operators.cu

```diff
@@ -367,6 +367,15 @@ std::shared_ptr<preprocessed_table> preprocessed_table::create(table_view const&
     new preprocessed_table(std::move(d_t), std::move(std::get<1>(null_pushed_table))));
 }
 
+two_table_comparator::two_table_comparator(const table_view& lhs,
+                                           const table_view& rhs,
+                                           rmm::cuda_stream_view stream)
+  : d_left_table(preprocessed_table::create(lhs, stream)),
+    d_right_table(preprocessed_table::create(rhs, stream))
+{
+  check_shape_compatibility(lhs, rhs);
+}
```

Comment:
For consistency:
```suggestion
two_table_comparator::two_table_comparator(table_view const& left,
                                           table_view const& right,
                                           rmm::cuda_stream_view stream)
  : d_left_table(preprocessed_table::create(left, stream)),
    d_right_table(preprocessed_table::create(right, stream))
{
  check_shape_compatibility(left, right);
}
```
https://github.com/rapidsai/cudf/blob/4429cc45e3ac9353b05b053e2bf4c22895b00926/cpp/src/table/row_operators.cu#L341-L350